lms/djangoapps/courseware/tests/test_discussion_xblock.py

```diff
@@ -304,6 +309,35 @@ def test_html_with_user(self):
         assert 'data-user-create-comment="false"' in html
         assert 'data-user-create-subcomment="false"' in html
 
+    @override_settings(DISCUSSION_MICROFRONTEND_URL="http://test.url")
+    @override_waffle_flag(ENABLE_DISCUSSIONS_MFE, True)
+    def test_embed_mfe_in_course(self):
```

Comment:
Let's test the xblock doesn't contain MFE UI when the flag is disabled.

Reply:
The existing tests are already doing that, since they make assertions about specific contents of block when the flag is disabled with default.

Reply:
sweet.